### PR TITLE
Client-side rendering

### PR DESCRIPTION
**Pull request by jamesstockman14**
I setup dynamic client-side rendering and refactored my seed script so that it works easily for my teammates.

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -2,43 +2,43 @@ import React from 'react';
...
-    // axios.get(urlWithId)
-    //   .then((fundingData) => {
-    //     this.setState({'itemId': fundingData.identifier});
-    //   })
-    //   .catch((err) => {
-    //     throw new Error(err);
-    //   });
+    let urlWithoutEndpoint = splitComponentUrl[0] + '//' + splitComponentUrl[2].slice(0, 12) + '04';
+    let endpoint = '/funding/' + splitComponentUrl[3];
+    console.log(urlWithoutEndpoint + endpoint);
```
Comment (wontkim) [inline client/src/index.jsx]: Remove console.log

File: database/index.js [hunk 2]
```diff
@@ -36,16 +35,23 @@ let headerSchema = {
...
-// });
-// SeedData.push(currentModel.update({upsert: true}));
-// }
-// Promise.all(SeedData);
+
+HeaderModel.find({})
+  .then((requestData) => {
+    if (requestData.length < 100) {
+      for (let i = 0; i < 100; i++) {
+        let generatedData = data.objectCreation(i);
```
Comment (wontkim) [inline database/index.js]: I forgot mongoose for the most part so can you tell me what is `data.objectCreation(i)` here? Where is `data` coming from and `objectCreation` as well?

**Review by wontkim — COMMENTED** [review]
Looks good James. Just had a few questions and then it's good to be merged

File: database/index.js (hunk 2 above)
Comment (jamesstockman14) [reply database/index.js]: data is the database file that I imported and objectCreation is a function from that file. I have updated the naming.

**Pull request merged**